src/scipp/html/formatting_html.py

```diff
@@ -126,19 +126,12 @@ def _short_data_repr_html_non_events(var, variances=False):
 
 
 def _short_data_repr_html_events(var):
-    if isinstance(var, sc.Dataset):
-        return str(var)
-    underlying = var.values[0]
```

Comment:
Could you also add a test that catches the segfault? e.g. the code snippet from the issue?

Reply:
There were test fails both in other PRs and on `main`.

Reply:
Strange it wasn't failing on my PR...